runtime/onert/core/src/util/shapeinf/Reshape.cc

```diff
@@ -75,25 +75,41 @@ void StaticInferer::visit(const ir::operation::Reshape &op)
     return;
   }
 
-  if (op.getInputs().size() == 1)
+  // New shape is given by second input tensor
+  if (op.getInputs().size() == 2)
```

Comment:
Does shape operand have a higher priority than op.param().new_shape?

Reply:
@ragmani Yes. According to tflite's `reshape` implementation, `input` has higher priority than `param` though it is different to https://www.tensorflow.org/lite/guide/ops_compatibility. As I understand, `tflite` seems to use `param` when the shape is determined before runtime, it uses `input` for dynamic shape. If `reshape` has `input`, `input` should be handled with high priority.